ui/views.py

```diff
@@ -261,12 +261,13 @@ def build_form(self, form_kwargs=None):
 
 
 @login_required
-def serve_media(request, media_path):
+def serve_static_assets(request, path):
     """
     View to serve media files in case settings.DEFAULT_FILE_STORAGE
     is django.core.files.storage.FileSystemStorage
     """
     # first check if the user has access to the file
+    media_path = os.path.join('assets', path)
```

Comment:
I suppose from STATIC_ASSET_BASEPATH in `learningresources/models.py` originally. I'll separate out the 'assets' prefix and reference it here and elsewhere I use it